app/models/document.rb

```diff
@@ -218,6 +218,15 @@ def publish!
         indexable_document.to_json,
       )
 
+      if self.update_type == "major"
+        email_alert_api.send_alert(
+          {
```

Comment:
This doesn't seem complete: don't we need the subject body of the email alert to be sent here? The existing app sends a whole lot of formatted HTML to email-alert-api.